pyspark/bigdl/nn/layer.py

```diff
@@ -4495,6 +4495,184 @@ class HardSigmoid(Layer):
     def __init__(self, bigdl_type="float"):
         super(HardSigmoid, self).__init__(None, bigdl_type)
 
+
+class Cropping2D(Layer):
+    """Cropping layer for 2D input (e.g. picture).
+
+    It crops along spatial dimensions, i.e. width and height.
+
+    # Arguments
+        cropping: int, or tuple of 2 ints, or tuple of 2 tuples of 2 ints.
+            - If int: the same symmetric cropping
+                is applied to width and height.
+            - If tuple of 2 ints:
+                interpreted as two different
+                symmetric cropping values for height and width:
+                `(symmetric_height_crop, symmetric_width_crop)`.
+            - If tuple of 2 tuples of 2 ints:
+                interpreted as
+                `((top_crop, bottom_crop), (left_crop, right_crop))`
+        data_format: A string,
+            one of `channels_last` (default) or `channels_first`.
```

Comment:
Use `nchw` and `nhwc` instead of `channels_last` and `channels_first`
See  SpatialConvolution for reference

Reply:
The mapping between bigdl and keras will be done later.